app/lib/submission_builder/ty2024/states/md/documents/md502.rb

```diff
@@ -54,7 +54,7 @@ def document
           xml.AddressLine1Txt sanitize_for_xml(@intake.permanent_street, 30)
           xml.AddressLine2Txt sanitize_for_xml(@intake.permanent_apartment, 30) if @intake.permanent_apartment.present?
           xml.CityNm sanitize_for_xml(@intake.permanent_city, 20)
-          xml.StateAbbreviationCd @intake.direct_file_data.mailing_state.upcase
+          xml.StateAbbreviationCd "MD"
```

Comment:
confirmed that although the DF _can_ send us the out of state `mailing_state` -- if the user confirms `:permanent_address_outside_md` it will lead the user to an ineligible screen. Which means that by the time we fill out the MD502, the user should've provided an address within MD (state hard coded on permanent address controller).